PRS10001-11000/PRS10194Yaeqob.xml

```diff
@@ -49,8 +49,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ያዕቆብ፡</persName>
                     <persName xml:lang="gez" corresp="#n1" type="normalized">Yāʿqob</persName>
-                    <persName><roleName type="title">aṣe</roleName></persName>
-                    <occupation type="ruler">apocryphal Aksumite king from king lists</occupation>
+                    <occupation type="ruler">king</occupation>
```

Comment:
apocryphal Aksumite king from king lists was better, I only asked to remove aṣe, there is no evidence he was ever referred to with the title. I provided the sequence of kings that can also be mentioned for even more clarity.